debezium-core/src/main/java/io/debezium/relational/history/KafkaDatabaseHistory.java

```diff
@@ -539,4 +541,10 @@ private Config getKafkaBrokerConfig(AdminClient admin) throws Exception {
         return configs.values().iterator().next();
     }
 
+    private static Validator forKafka(final Validator validator) {
```

Comment:
This feels really hacky, but given the static context of all this, I cannot see any good solution either (like adding only those fields relevant for the actually used history impl). How comes the isRequired check wasn't a problem before?

Reply:
Because it was in configDef only but not in ALL_FIELDS and validation is based on ALL_FIELDS. MySQL does not have the same problem exactly due to the same reason.

Reply:
So something is still broken. MySQL uses `EXPOSED_FIELDS` for validation, and there the kafka history props are contained. Indeed a test is failing now as of this PR (see Travis), as the expected failure isn't raised anymore. I'll adjust this test accordingly.